WD-Firmware/rtos/rtosal/api_inc/rtosal_config.h

```diff
@@ -26,15 +26,16 @@
 /**
 * include files
 */
+#ifdef D_USE_FREERTOS
+   #include "FreeRTOSConfig.h"
```

Comment:
1. put this include at the end of the file
2. remove all queries.
3. in freeRtosConfig put #ifndef -> #define

Reply:
done https://github.com/westerndigitalcorporation/riscv-fw-infrastructure/pull/12/commits/6c26f8aba68f07621c53b564702cbab8a7d91dd1